dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/business/ContentletAPI.java

```diff
@@ -1148,8 +1148,24 @@ public Contentlet checkin(Contentlet currentContentlet, ContentletRelationships
 	 * @throws DotDataException
 	 * @throws DotStateException if the identifier is for contentlet
 	 */
+
 	public List<Contentlet> findAllVersions(Identifier identifier, User user, boolean respectFrontendRoles) throws DotSecurityException, DotDataException, DotStateException;
-	
+
+	/**
+	 * Retrieves all versions for a contentlet identifier
+	 * Note this method should not be used currently because it could pull too many versions.
```

Comment:
I think we should change the javadoc explaining what the `bringOldVersions` does